superset/thumbnails/tasks.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This file is really the same as `superset/tasks/thumbnails.py` before, just updated with the new logic (fetching the executor and overriding the username etc)